modules/rosa-getting-support.adoc

```diff
@@ -0,0 +1,25 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// support/rosa-getting-support.adoc
+
+[id="rosa-getting-support_{context}"]
+= Getting support for {product-title}
+
+[role="_abstract"]
+Learn about the Support options for {product-title}.
+
+If you experience difficulty with a procedure described in this documentation, visit the link:https://access.redhat.com/[Red Hat Customer Portal]. Through the Customer Portal, you can:
+
+* Search or browse through the Red Hat Knowledgebase of technical support articles about Red Hat products.
+* Access other product documentation.
+* Submit a support case to Red Hat Support:
+.. Click *Open a New Case*.
+.. Select the reason for the support ticket, such as `Defect/Bug` or `Account/Customer Service Request`.
+.. In the `Product` field, enter `OpenShift` to filter the list. Select `{product-title}` and the version from the drop-down menus.
+.. Complete the remaining fields.
+.. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
+
+You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support].
+
+If you have a suggestion for improving this documentation or have found an error, submit a link:https://bugzilla.redhat.com/index.cgi[Bugzilla] report against the OpenShift Container Platform product for the Documentation component. Be sure to provide specific details, such as the section name and {product-title} version.
```

Comment:
Is that really where you want customers to file issues against managed services docs?  Directly in BZ?  Shouldn't it go through the same flow described earlier in this PR (RH customer portal or AWS support).

Reply:
Paying customers are normally funneled through support, but there is nothing stopping them from taking any other path.  However, if it has caused them some issue there is little way to prioritize that back to an incident if they created a BZ directly.

TL;DR is that it definitely benefits the customer to open a support case, if a piece of the docs causing them any issue.

Reply:
I don't want customers to ever go to BZ.  I don't think we should document anything other than "call support".

Reply:
@jeremyeder  @davemulford  This Bugzilla info mirrors the OSD info. Just FYI. It's your call. LMK